core/src/main/java/com/linecorp/armeria/server/RoutingStatus.java

```diff
@@ -36,12 +36,7 @@ public enum RoutingStatus {
     /**
      * An {@code "OPTIONS * HTTP/1.1"} request.
      */
-    OPTIONS(false),
-
-    /**
-     * The request specified an invalid path.
-     */
-    INVALID_PATH(false);
```

Comment:
We reject an invalid path earlier now so the router doesn't even need to deal with it.